### DISCUSSION TITLE
Replace twxs.cmake with something up to date

### DISCUSSION THREAD

**Discussion by rafrafek** [microsoft/vscode-cpptools#12140] [Ideas]
Hello,

Currently the official vscode-cpptools pack contains twxs.cmake extension:

https://marketplace.visualstudio.com/items?itemName=twxs.cmake

but the last commit made to that extension is from 2019:

https://github.com/twxs/vs.language.cmake

Can the extension be replaced with something more up to date?

**Comment by bobbrow** [comment]
We have to defer to the CMake Tools extension for this one.  I see it in their milestone to add language services to the extension so that it doesn't need to take a dependency on the older extension.  I don't believe the dependency on twxs.cmake is a hard dependency - which means you can uninstall it.  There was another language server extension for CMake that was pretty good; you can try it out and see if you like it better: https://marketplace.visualstudio.com/items?itemName=josetr.cmake-language-support-vscode

**Reply by rafrafek** [reply]
Thank you for your reply! 🙏
